spec/lib/delegates/wa/cantwell_spec.rb

```diff
@@ -0,0 +1,38 @@
+# CHANGE MESSAGE IF USING AGAIN!
+#
+# VCR cannot record phantomjs requests, so do not ever set this
+# to run during any kind of automated build system. Use sparingly
+# while developing as needed.
+
+# require 'rails_helper'
+
+# describe Delegates::WA::Cantwell do
```

Comment:
Use either RSpec tags to exclude this from the default run list, or surround it with an `if ENV['SLOW_TESTS']` block.